### PR TITLE
BIGTOP-2523. Incorrect cycle dependency during puppet deployment causes error at Debian 8

### PR DESCRIPTION
**Pull request by kirill-morozov-epam**
…ppet repo
https://issues.apache.org/jira/browse/BIGTOP-2523
Incorrect cycle dependency during puppet deployment causes error at Debian 8

### CODE REVIEW

File: bigtop_toolchain/bin/puppetize.sh [hunk 1]
```diff
@@ -54,8 +54,13 @@ case ${ID}-${VERSION_ID} in
...
 	;;
     debian-8*)
+    apt-get update
+	apt-get -y install wget
+	# BIGTOP-2523. in order to install puppet 3.8 we need to get it from puppet repo
+	wget -O /tmp/puppetlabs-release-trusty.deb https://apt.puppetlabs.com/puppetlabs-release-trusty.deb && dpkg -i /tmp/puppetlabs-release-trusty.deb
+	rm -f /tmp/puppetlabs-release-trusty.deb
 	apt-get update
-	apt-get -y install curl sudo unzip wget puppet
+    apt-get -y install curl sudo unzip puppet
```
Comment (c0s) [inline bigtop_toolchain/bin/puppetize.sh]: this looks like a formatting change, irrelevant to the scope

**Comment by rvs** [comment]
LGTM! (aside from the formatting fix). Olaf taking a look would be great, but I think it is small enough so we can just go ahead and commit it.

**Comment by c0s** [comment]
I am confused. Who's the author of the patch? The commit shows
```
From: Artur Papikyan <artur_papikyan@epam.com>
Date: Fri, 26 Aug 2016 16:12:42 +0300
```
Is this correct?

**Comment by kirill-morozov-epam** [comment]
This is correct. Artur the author of the patch

**Comment by c0s** [comment]
I have committed the fix. Thanks for your help guys, please close the PR as you see fit.

**Pull request closed without merging**